PORT-9832 Update Multi-account minimum permission to install the Ocean AWS Exporter (#1535)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/installations/multi_account.md

<CenterRoundedImage imgSrc='/img/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/aws-single-step-2.jpg' />

> First, we will create a role with necessary permissions (If you're running the terraform installation, this will be done for you). We'll call this role `ReadOnlyPermissionsOceanRole`, with `I can read resources in this account` policy.

Reviewer: ```suggestion
First, we will create a role with necessary permissions (if you're running the terraform installation, this will be done for you). We'll call this role `ReadOnlyPermissionsOceanRole`, with `I can read resources in this account` policy.
```

:::tip
The name of this role (not the ARN) is referenced as `integration_account` in this doc.

> If you need to read from the integration account you'll need to add CloudControl (cloudformation) and the resources permissions
> in this example it's S3 bucket permissions

Reviewer: ```suggestion
If you need to read from the `integration account`, you'll need to add CloudControl (cloudformation) and the resources permissions. In this example, we refer to the S3 bucket permissions
```

In order to keep adding accounts to the integration's scope, permissions must be delivered for and from each of the accounts.
For each account you want to have, you should make sure the following applies:

> In each non-root account (target member account), The Role `accountReadRoleName` must exist with necessary permissions to access S3 resources, list regions, and interact with CloudFormation resources, and include `accountReadRoleName` from the integration account in its trust policy. See [reference](#member-account)

Reviewer: ```suggestion
In each non-root account (target member account), The Role `accountReadRoleName` must exist with the necessary permissions to access S3 resources, list regions, interact with CloudFormation resources, and include `accountReadRoleName` from the `integration account` in its trust policy. See [reference](#member-account)
```
